Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java

```diff
@@ -130,6 +133,14 @@ public void propertyChange(PropertyChangeEvent evt) {
                 jobProcessingExecutor.submit(new DataSourceNameChangedTask(dbManager, evt));
             }
             break;
+            case OS_ACCT_INSTANCES_ADDED: {
+                // STUB, TO BE REPLACED
```

Comment:
SHould this comment be removed

Reply:
@markmckinnon, my intention was for you to remove it and replace my logging with your code. Does that work for you?

Reply:
Yes it does.

Reply:
Cool.